graviti/manager/dataset.py

```diff
@@ -41,11 +37,44 @@ class Dataset(Mapping[str, DataFrame]):  # pylint: disable=too-many-instance-att
         dataset_id: Dataset ID.
         name: The name of the dataset, unique for a user.
         alias: Dataset alias.
+        default_branch: The default branch of dataset.
         commit_id: The commit ID.
-        branch_name: The branch name.
+        created_at: The time when the dataset was created.
+        updated_at: The time when the dataset was last modified.
+        owner: The owner of the dataset.
+        is_public: Whether the dataset is public.
+        config: The config of dataset.
 
     """
 
+    _T = TypeVar("_T", bound="Dataset")
+
+    _repr_type = ReprType.MAPPING
+    _repr_attrs: Tuple[str, ...] = (
+        "dataset_id",
+        "alias",
+        "default_branch",
+        "commit_id",
+        "created_at",
+        "updated_at",
+        "is_public",
+        "config",
+        "branch",
+    )
+
+    access_key: str = attr()
+    url: str = attr()
+    dataset_id: str = attr(key="id")
```

Comment:
we can assume that the dataset id is useless for user, shall we make it private and remove it from the _repr_attrs

Reply:
ok